[CI] Failure in MixedClusterClientYamlTestSuiteIT indices.stats/20_translog/Translog stats on closed indices with soft-deletes

**Build scan**:

https://gradle-enterprise.elastic.co/s/g637ms6cz3nvc

**Repro line**:

```
./gradlew ':qa:mixed-cluster:v7.8.1#mixedClusterTest' --tests "org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" \
  -Dtests.method="test {p0=indices.stats/20_translog/Translog stats on closed indices with soft-deletes}" \
  -Dtests.seed=859ADC1A3F80198E \
  -Dtests.security.manager=true \
  -Dtests.locale=fr-CA \
  -Dtests.timezone=America/Tegucigalpa \
  -Dtests.distribution=default \
  -Druntime.java=8
```

**Reproduces locally?**:

No

**Applicable branches**:

7.8

**Failure history**:

https://build-stats.elastic.co/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-30d,mode:quick,to:now))&_a=(columns:!(_source),index:b646ed00-7efc-11e8-bf69-63c8ef516157,interval:auto,query:(language:lucene,query:'MixedClusterClientYamlTestSuiteIT%20AND%2020_translog%20AND%20soft-deletes'),sort:!(process.time-start,desc))

**Failure excerpt**:

Assertion failure is:

```
java.lang.AssertionError: Failure at [indices.stats/20_translog:273]: field [acknowledged] doesn't have a true value
	Expected: not a string equal to "false" ignoring case
	     but: was "false"
```

Output is:

```
[2020-08-10T07:18:36,912][INFO ][o.e.b.MixedClusterClientYamlTestSuiteIT] [test] [p0=indices.stats/20_translog/Translog stats on closed indices with soft-deletes] before test
[2020-08-10T07:18:37,226][INFO ][o.e.b.MixedClusterClientYamlTestSuiteIT] [test] Stash dump on test failure [{
  "stash" : {
    "body" : {
      "acknowledged" : false,
      "shards_acknowledged" : false,
      "indices" : {
        "test" : {
          "closed" : false,
          "failedShards" : {
            "0" : {
              "failures" : [
                {
                  "node" : "cHTHt6wERIqlWQj60Ni-yg",
                  "shard" : 0,
                  "index" : "test",
                  "status" : "INTERNAL_SERVER_ERROR",
                  "reason" : {
                    "type" : "illegal_state_exception",
                    "reason" : "Global checkpoint [-1] mismatches maximum sequence number [0] on index shard [test][0]"
                  }
                }
              ]
            }
          }
        }
      }
    }
  }
}]
[2020-08-10T07:18:37,305][INFO ][o.e.b.MixedClusterClientYamlTestSuiteIT] [test] Clearing template [global]
[2020-08-10T07:18:37,327][INFO ][o.e.b.MixedClusterClientYamlTestSuiteIT] [test] Clearing template [ilm-history]
[2020-08-10T07:18:37,422][INFO ][o.e.b.MixedClusterClientYamlTestSuiteIT] [test] [p0=indices.stats/20_translog/Translog stats on closed indices with soft-deletes] after test
```

The failure is the same as in #46535, but presumably the root cause is different as that was fixed a long time ago.

--- a/rest-api-spec/src/main/resources/rest-api-spec/test/indices.stats/20_translog.yml
+++ b/rest-api-spec/src/main/resources/rest-api-spec/test/indices.stats/20_translog.yml
@@ -191,6 +191,7 @@
         body:
           settings:
             soft_deletes.enabled: false
+            routing.rebalance.enable: "none" # prevents shard relocations while we are closing an index
       warnings:
         - Creating indices with soft-deletes disabled is deprecated and will be removed in future Elasticsearch versions.
           Please do not specify value for setting [index.soft_deletes.enabled] of index [test].
@@ -248,6 +249,7 @@
         body:
           settings:
             soft_deletes.enabled: true
+            routing.rebalance.enable: "none" # prevents shard relocations while we are closing an index
   - do:
       cluster.health:
         wait_for_no_initializing_shards: true

